### PR TITLE
1.7 release notes Updated

### PR DESCRIPTION
**Pull request by resmaa**
Revisited the release notes as per review feed back

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/301#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @resmaa. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: packages/docs/content/13-release-notes.md [hunk 1]
```diff
@@ -13,6 +13,25 @@ import InfoBox from '@librarium/shared/src/components/InfoBox';
 import PointsOfInterest from '@librarium/shared/src/components/common/PointOfInterest';
 import Tooltip from "@librarium/shared/src/components/ui/Tooltip";
 
+# January 07, 2021 - Release 1.7.0
```
Comment (zulfilee) [inline packages/docs/content/13-release-notes.md:16]: Date is February

File: packages/docs/content/13-release-notes.md [hunk 2]
```diff
@@ -13,6 +13,25 @@ import InfoBox from '@librarium/shared/src/components/InfoBox';
...
+The release has also come up with added Brownfield support.
+
+Upgradations, Enhancements and Modifications:
+
+- The cluster API CAPI updated to 0.3.12.
+- CAPZ, the cluster API provider for Azure platform is updated to 0.4.11.
+- Upgraded with multiple profile support to clusters.
+- New node agent service supports an ansible playbook to perform node related tasks.
+- Ansible OS-patch playbook inclusion for security patch in nodes.
+- Spectro Cloud is enabling legacy brown field support bringing any cloud to spectro
```
Comment (zulfilee) [inline packages/docs/content/13-release-notes.md:28]: brownfield is one word

File: packages/docs/content/13-release-notes.md [hunk 3]
```diff
@@ -13,6 +13,25 @@ import InfoBox from '@librarium/shared/src/components/InfoBox';
 import PointsOfInterest from '@librarium/shared/src/components/common/PointOfInterest';
 import Tooltip from "@librarium/shared/src/components/ui/Tooltip";
 
+# January 07, 2021 - Release 1.7.0
+
+We are pleased to announce the release of Spectro Cloud Version 1.7.0 focusing on a dedicated User Interface which eases the migration from a quick start version to enterprise version.
+The release has also come up with added Brownfield support.
```
Comment (zulfilee) [inline packages/docs/content/13-release-notes.md:19]: The release also included Brownfield support

**Review by zulfilee — CHANGES_REQUESTED** [review]
Take care of the points

**Review by zulfilee — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/301#" title="Author self-approved">resmaa</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/301#pullrequestreview-589290729" title="Approved">zulfilee</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**